src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,157 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        Storage.people.clear();
+    }
+
+    @Test
+    void registerUser_WithLoginNull_NotOk() {
+        user.setLogin(null);
+        user.setPassword("valid123");
+        user.setAge(20);
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void registerUser_WithInvalidLoginLength_NotOk() {
+        user.setLogin("Steve");
+        user.setPassword("valid123");
+        user.setAge(20);
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void registerUser_WithPasswordNull_NotOk() {
+        user.setLogin("Steve123");
+        user.setPassword(null);
+        user.setAge(20);
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void registerUser_WithInvalidPasswordLength_NotOk() {
+        user.setLogin("Steve123");
+        user.setPassword("qwert");
+        user.setAge(20);
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(user);
+        });
+    }
```

Comment:
The test `registerUser_WithInvalidPasswordLength_NotOk` is good as it checks for a password that is too short, but make sure to also include a test for passwords that are just short enough to meet the minimum length requirement (edge case).